### PR TITLE
Remove RxTextFormFieldBuilder suffix icon padding

### PR DESCRIPTION
**Pull request by JvnSlv**
#### Overview

This PR should fix the padding of the suffix icon of RxTextFormFieldBuilder.
The bug occurs when the RxTextFormFieldBuilder is used with the TextField or TextFormField that has a specified height of under 50 pixels. 

Screenshots of before and after are included ⬇️ 

Screenshot_1707303329
Screenshot_1707303338

#### Checklist

*Implementation*
- [ ] ~~Implementation matches ticket acceptance criteria and technical notes~~
- [x] Manually tested against Acceptance Criteria
- [ ] ~~UI checked in Light / Dark mode~~

*Stability*
- [ ] ~~Checked if changes affect any features and verified affected features work as expected~~
- [ ] ~~Added unit tests for new code and verified existing tests work as expected~~

*Code quality*
- [x] Updated `CHANGELOG.md`, `README.md` and package versions in `pubspec.yaml`
- [x] Dependencies are updated to latest versions or new tickets are created if there are breaking changes or deprecations
- [x] If an unrelated part of the codebase needs to be updated or refactored, create tickets with proposed changes

### CODE REVIEW

**Review by DDavidPrime — APPROVED** [review]

**Comment by StanevPrime** [comment]
@JvnSlv, thank you for this PR. Could you please sure you address the `Code quality` section ?

**Review by StanevPrime — CHANGES_REQUESTED** [review]
Could you please sure you address the Code quality section ?

File: packages/flutter_rx_bloc/pubspec.yaml [hunk 1]
```diff
@@ -11,18 +11,18 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  provider: ^6.0.5
+  provider: ^6.1.1
```
Comment (StanevPrime) [inline packages/flutter_rx_bloc/pubspec.yaml:14]: Do we really need to upgrade to 6.1.1?

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: packages/flutter_rx_bloc/pubspec.yaml (hunk 1 above)
Comment (JvnSlv) [reply packages/flutter_rx_bloc/pubspec.yaml:14]: We don't, but it's the latest version

Comment (JvnSlv) [reply packages/flutter_rx_bloc/pubspec.yaml]: I guess no.
I have reverted all the dependencies upgrades

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**